server/src/rest/api.rs

```diff
@@ -966,11 +1085,12 @@ pub async fn update_aircraft(
                 schedule: payload.schedule.unwrap_or(vehicle.schedule),
                 last_maintenance: if let Some(last_maintenance) = payload.last_maintenance {
                     if let Some(last_maintenance) = last_maintenance {
-                        let time_stamp_result = Timestamp::from_str(&last_maintenance);
-                        if time_stamp_result.is_ok() {
-                            Some(time_stamp_result.unwrap())
-                        } else {
-                            None
+                        match Timestamp::from_str(&last_maintenance) {
```

Comment:
this is a little messy (too many nests) can you pull this block out of the `match` statement (above it)?

Same with the one below, then this PR is good to go

Reply:
[187d086](https://github.com/Arrow-air/svc-assets/pull/16/commits/187d086d04cf2fe60ee4a441147a9992d194b40e)